ui/src/app/components/source-details-form/source-details-form.component.ts

```diff
@@ -54,6 +61,7 @@ export class SourceDetailsFormComponent implements OnInit {
       },
       error: (err: any) => {
         this.errorMsg = err.error
+        console.log(this.errorMsg)
```

Comment:
Where does this log to?

Reply:
This was debugging. Removed it from the PR